packages/common/core-interfaces/src/provider.ts

```diff
@@ -85,3 +85,19 @@ export type FluidObject<T = unknown> = {
  *
  */
 export type FluidObjectKeys<T> = keyof FluidObject<T>;
+
+/**
+ * @public
+ */
+export const FluidStaticEntryPoint = Symbol.for("FluidStaticEntryPoint");
+/**
+ * @public
+ */
+export type FluidStaticEntryPoint = typeof FluidStaticEntryPoint;
+
+/**
+ * @public
+ */
+export interface FluidStaticEntryPointProvider {
```

Comment:
we use the symbol, and the provider pattern to decouple what the symbol provides to not leak types we don't want to leak.